src/AcsfLogs.php

```diff
@@ -50,13 +52,18 @@ public function getRootLogsFolder()
    * @return string
    * @throws \Exception
    */
-  public function getLogsFolder($iteration = 0, $createFolder = TRUE) {
+  public function getLogsFolder($iteration = 0, $createFolder = TRUE, $date = NULL) {
```

Comment:
- use the get lastLogsfolder to find the very last.
- if it has a finish marker -> return the folder
- if not -> then create next iteration folder